packages/client/hmi-client/src/components/widgets/tera-edit-value.vue

```diff
@@ -0,0 +1,48 @@
+<template>
```

Comment:
I added this as a component instead of us creating it everytime which I found annoying...basically an edit field with the ability to confirm or cancel the changes.  We can pass in the component type we want to use as well (h1,h2,h3...)
<img width="322" alt="Screenshot 2024-07-02 at 6 25 04 PM" src="https://github.com/DARPA-ASKEM/terarium/assets/33158416/d9f0a4aa-1777-4f51-9c26-9093e8ccf9be">

Reply:
Lets maybe have a quickie chat here, cause there are a handful of edit-inputbox-ish component and we need a (better) standard on which one is which.

Reply:
We will rename this component.

Reply:
I've renamed this to `tera-toggleable-edit` 😛